legacy/signals/signals.go

```diff
@@ -24,31 +24,68 @@ import (
 	"github.com/sirupsen/logrus"
 )
 
+var (
+	// ExitSignals are used to determine if an incoming os.Signal should cause termination.
+	ExitSignals map[os.Signal]bool = map[os.Signal]bool{
+		syscall.SIGHUP:  true,
+		syscall.SIGINT:  true,
+		syscall.SIGABRT: true,
+		syscall.SIGILL:  true,
+		syscall.SIGQUIT: true,
+		syscall.SIGTERM: true,
+		syscall.SIGSEGV: true,
+		syscall.SIGTSTP: true,
+	}
+	// ExitChannel is for gracefully terminating the LogSignals() function.
+	ExitChannel chan bool = make(chan bool, 1)
+	// SignalChannel is for passing observed OS signals.
```

Comment:
Nit: I think SignalChannel is for just listening to signals from the OS. "passing" sounds like we (auditor) is doing the passing, which is not true (we always only receive from the end of this channel; we never put (pass) things into it).

Reply:
Ack.